docs/installation.rst

```diff
@@ -57,22 +61,10 @@ This will create a ``dist`` directory in ``treeherder-ui`` where concatenated an
 Configuration
 =============
```

Comment:
I've moved higher up & specifically referenced the "file created above", but I think it still should be separate from "running the server". The first section is a basic "how to get running with the defaults as fast as possible", the later configuration section is optional.